### DISCUSSION TITLE
Where to manually put the sd_xl_base.yaml & sd_xl_refiner.yaml?  Can't automatically download due to internet problems

### DISCUSSION THREAD

**Discussion by minienglish1** [vladmandic/sdnext#1957] [Q&A] [answered]
11:42:21-594381 ERROR    Diffusers failed loading model using pipeline:
                         C:\Users\3090\SD.Next\models\Stable-diffusion\sd_xl_base_1.0.safetensors Stable Diffusion XL
                         HTTPSConnectionPool(host='raw.githubusercontent.com', port=443): Max retries exceeded with
                         url: /Stability-AI/generative-models/main/configs/inference/sd_xl_base.yaml (Caused by
                         SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify
                         failed: unable to get local issuer certificate (_ssl.c:1007)')))
11:42:21-617824 WARNING  Model not loaded

As shown above, I get SSL Errors when SD.Next tries to download the files.  This is due to the country I'm in having restricted internet.

I have downloaded them manually, but I don't know where to put them.  I tried putting them with the safetensors files in models\Stable-Diffusion and in configs, but didn't fix the problem.

Thanks in advance.

**Comment by vladmandic** [comment, marked as answer]
They will be included in the near future to allow fully offline work, but at the moment it's online only.

**Reply by minienglish1** [reply]
That explains a lot.  Much appreciate you taking the time to answer!  Thanks for all your hard work!
